admin-manual/installation/installation.rst

```diff
@@ -6,290 +6,246 @@ Installation
-This has been introduced to allow one central place to store packages for multiple operating systems.
+.. _install-pkg-ubuntu:
 
-There are some dependencies still hosted on Launchpad, that have not yet been migrated to packages.archivematica.org.
-In a future release, all the requirements will be hosted in one repository, for the time being it is necessary to set up
-two different sources of packages.
+Installing Ubuntu Packages
+--------------------------
 
-1. Add the archivematica/externals PPA to your list of trusted repositories (if add-apt-repositories is not available you must install python-software-properties first):
+Archivematica packages are hosted at packages.archivematica.org. This has been 
+introduced to allow one central place to store packages for multiple OS's.
 
-.. code:: bash
+1. Add Package Sources
 
-   sudo apt-get update
-   sudo apt-get install python-software-properties
-   sudo add-apt-repository ppa:archivematica/externals
+* Archivematica
 
-2. Add packages.archivematica.org to your list of trusted repositories
+Add packages.archivematica.org to your list of trusted repositories
 
 .. code:: bash
 
    sudo wget -O - https://packages.archivematica.org/1.6.x/key.asc | sudo apt-key add -
    sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.6.x/ubuntu trusty main" >> /etc/apt/sources.list'
    sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.6.x/ubuntu-externals trusty main" >> /etc/apt/sources.list'
 
-3. Add the ElasticSearch apt repository next:
+* ElasticSearch
+
+Elasticsearch comes from its own package repository
 
 .. code:: bash
 
    sudo wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | sudo apt-key add -
    sudo sh -c 'echo "deb http://packages.elasticsearch.org/elasticsearch/1.7/debian stable main" >> /etc/apt/sources.list'
 
-4. Update your system to the most recent 14.04 release. This step will also fetch a list of the software from the PPAs you just added to your system.
+2. Update your system 
+
+Update to the most recent 14.04 release. This step will also fetch a list of 
+the software from the PPAs you just added to your system.
 
 .. code:: bash
 
    sudo apt-get update
    sudo apt-get upgrade
 
-5. Install Elasticsearch
+3. Install Elasticsearch
 
 .. code:: bash
 
    sudo apt-get install elasticsearch
 
-6. Install the storage service package
+4. Install the storage service package
 
 .. code:: bash
 
    sudo apt-get install -y archivematica-storage-service
 
-7. Install pip
+5. Install pip.  
+
+This is used to install python dependencies for both the storage service and 
+the dashboard.  There is a _known issue: https://bugs.launchpad.net/ubuntu/+source/python-pip/+bug/1658844 with the version of pip installed on 
+Ubuntu 14.04, which makes this step necessary.
 
 .. code:: bash
 
    wget -O /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py
```

Comment:
al references to get-pip should be now

apt-get install -y python-pip
pip install -U setuptools